usr/share/rear/prep/default/310_include_uefi_tools.sh

```diff
@@ -5,10 +5,13 @@ if grep -qw 'noefi' /proc/cmdline; then
     return
 fi
 
-# next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it)
-if [[ $(find /boot -maxdepth 1 -iname efi -type d | wc -l) -eq 0 ]] ; then
-    return    # must be mounted
-fi
+# Next step, is checking /boot/efi directory case-insensitive for the /EFI part (we need it).
+# If no /boot/[eE][fF][iI] directory can be found we cannot copy the UEFI binaries we might need.
```

Comment:
@ProBackup-nl 
many thanks for your explanations!
I cannot really help here (currently I have neither time
nor sufficient knowledge) but I appreciate adaptions
and enhancements that make it work more generally.
Ideally it may get also more fail-safe preferaby that
it errors out during "rear mkrescue" if mandatory
things fail or are missing so that users see in advance
(i.e. during "rear mkrescue") that later "rear recover"
would not work for them.